src/main.py

```diff
@@ -234,6 +238,50 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(
+        description="The uploaded file for submission."
+    ),
+    submit_message: str = SlashOption(
+        required=False, description="The text for submission message."
+    ),
+) -> None:
+    """Send a message to user with embedded preview of uploaded song's metadata."""
+    user = interaction.user
+    attachment_filepath = filepath_builder(interaction.id, uploaded_file)
+    random_emoji = random.choice(config.emoji_list)
+    await uploaded_file.save(fp=attachment_filepath)
+
+    if is_valid_media(uploaded_file.content_type) is False:
+        await interaction.response.send_message(
+            "Sorry, looks like you didn't send the correct media type. \nTry that one again, kid.",
+            ephemeral=True,
+        )
+    else:
```

Comment:
To avoid unnecessary indent I'd just make it return in the if statement and have no else